tests/low-disk/create_and_search_items.py

```diff
@@ -0,0 +1,90 @@
+import argparse
+import random
+import uuid
+
+import requests
+
+
+def generate_points(amount: int):
+    for _ in range(amount):
+        result_item = {"points": []}
+        for _ in range(100):
+            result_item["points"].append(
+                {
+                    "id": str(uuid.uuid4()),
+                    "vector": [round(random.uniform(0, 1), 2) for _ in range(4)],
+                    "payload": {"city": ["Berlin", "London"]}
+                }
+            )
+        yield result_item
+
+
+def create_collection(qdrant_host, collection_name):
+    resp = requests.put(
+        f"{qdrant_host}/collections/{collection_name}?timeout=60", json={
+            "vectors": {
+                "size": 4,
+                "distance": "Cosine"
+            }
+        })
+    if resp.status_code != 200:
+        print(f"Collection creation failed with response body:\n{resp.json()}")
+        exit(-1)
+
+
+def insert_points(qdrant_host, collection_name, batch_json):
+    resp = requests.put(
+        f"{qdrant_host}/collections/{collection_name}/points?wait=true", json=batch_json
+    )
+    EXPECTED_ERROR_MESSAGE = "No space left on device"
+    if resp.status_code != 200:
+        if resp.status_code == 500 and EXPECTED_ERROR_MESSAGE in resp.text:
+            requests.put(f"{qdrant_host}/collections/{collection_name}/points?wait=true", json=batch_json)
+        else:
+            error_response = resp.json()
+            print(f"Points insertions failed with response body:\n{error_response}")
+            exit(-2)
+
```

Comment:
Either `status_code:200` or `status_code:500 + EXPECTED_ERROR_MESSAGE` is expected